### PR TITLE
Update LEGO Batman

### PR DESCRIPTION
**Pull request by Biksell**
If you are adding or modifying an Auto Splitter, please fill out this checklist:
- [x] My Auto Splitter does not contain any malicious functionality and I'm entirely responsible for any damages myself. (Any kind of abuse in an Auto Splitter will result in an immediate ban.)
- [x] I am not replacing an existing Auto Splitter by a different author, or I have received permission from the author to replace the existing Auto Splitter.
- [x] The Auto Splitter has an open source license that allows anyone to fork and host it.

To note:
Prior repository's README states that the scripts are no longer maintained and can be replaced. The old address in the script was inconsistent so current script was built from scratch.

### CODE REVIEW

**Comment by just-ero** [comment]
@drtchops Are you alright with this change?

**Comment by Biksell** [comment]
From what I've been told the person in question has been inactive in the community for 2 years, but I've just contacted them via speedrun.com messages if the statement in their repository isn't enough to justify this replacement.

File: LiveSplit.AutoSplitters.xml [hunk 1]
```diff
@@ -4760,11 +4760,11 @@
             <Game>Lego Batman: The Videogame</Game>
         </Games>
         <URLs>
-            <URL>https://raw.githubusercontent.com/drtchops/asl/master/legobatman.asl</URL>
+            <URL>https://raw.githubusercontent.com/Biksell/asl/main/LEGO%20Batman/LEGOBatman.asl</URL>
         </URLs>
         <Type>Script</Type>
-        <Description>Load Removal is available. (By DrTChops)</Description>
-        <Website>https://github.com/drtchops/asl/blob/master/README.md</Website>
+        <Description>Autosplitting and Load removal is available. (By Siedemnastek & Biksel)</Description>
```
Comment (wooferzfg) [inline LiveSplit.AutoSplitters.xml:4766]:
```suggestion
        <Description>Autosplitting and Load removal is available. (By Siedemnastek &amp; Biksel)</Description>
```

**Comment by wooferzfg** [comment]
DrTChops gives permission in their readme:

> These scripts are no longer maintained. If something needs to be updated, fork the repo or copy the script somewhere else, host it yourself, and point livesplit to the new version.

**Comment by drtchops** [comment]
👍 

> but I've just contacted them via speedrun.com messages if the statement in their repository isn't enough to justify this replacement.

I don't have an account there so idk who you messaged, lol

**Comment by Biksell** [comment]
> 👍
> 
> > but I've just contacted them via speedrun.com messages if the statement in their repository isn't enough to justify this replacement.
> 
> I don't have an account there so idk who you messaged, lol

Oh apologies, I wasn't around in the community back when you made the script, so I was just going off of information that was relayed to me and they must've mistaken you for someone else 😅

**Pull request merged**